packages/amplication-data-service-generator/src/server/static/scripts/customSeed.ts

```diff
@@ -1,4 +1,4 @@
-import { PrismaClient } from "@prisma/client";
+import { PrismaClient } from "../prisma/generated-prisma-client";
```

Comment:
this code will never be updated in existing apps because this file is ignored. we need to think how to convert this to the user (docs?)

Reply:
I will write a description for the release notes and the docs